opm/parser/share/keywords/000_Eclipse100/G/GCONPROD

```diff
@@ -1,10 +1,10 @@
 {"name" : "GCONPROD" , "sections" : ["SCHEDULE"], "items" : [
         {"name" : "GROUP" , "value_type" : "STRING" },
         {"name" : "CONTROL_MODE" , "value_type" : "STRING" , "default" : "NONE"},
-        {"name" : "OIL_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : 0},
-        {"name" : "WATER_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : 0},
-        {"name" : "GAS_TARGET" , "value_type" : "DOUBLE" , "dimension" : "GasSurfaceVolume/Time", "default" : 0},
-        {"name" : "LIQUID_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : 0},
+        {"name" : "OIL_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : -999e100},
```

Comment:
The default in these cases should not be zero, but "no rate at all". I agree that this must be handled eventually with the defaultApplied(), but the default equal to zero is misleading, and an invalid rate is therefore better.

Since in JSON these things can be strings, could we use "default" : "no rate" and treat that somehow? I am not familiar with how the JSON is actually parsed and used, so it may not be feasible.